drivers/bluetooth/hci/Kconfig

```diff
@@ -92,3 +98,18 @@ config BT_SPI_BLUENRG
 	  Stack. Current driver supports: ST BLUENRG-MS.
 
 endif # BT_SPI
+
+if BT_RPMSG
+
+config BT_RPMSG_NRF53
```

Comment:
I would like to see a minor help text here explaining the channel configuration

Reply:
added short description